pkgs/tools/misc/sensible-utils/default.nix

```diff
@@ -0,0 +1,52 @@
+{ stdenv, lib, fetchgit, makeWrapper, bash }:
+
+let
+  pname = "sensible-utils";
+  version = "0.0.18";
+in
+stdenv.mkDerivation rec {
+  inherit pname version;
+
+  # https://salsa.debian.org/debian/sensible-utils
+  src = fetchgit {
```

Comment:
We can't use fetchFromGitLab?

Reply:
Ah, you can enter the "domain". Thank you!